libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -371,21 +374,22 @@ def deploy_cortx_cluster(self, sol_file_path: str, master_node_list: list,
             each.join()
 
         self.prereq_git(master_node_list[0], git_tag)
-        self.checkout_update_deploy_script(master_node_list[0], self.git_script_tag)
         self.copy_sol_file(master_node_list[0], sol_file_path, self.deploy_cfg["k8s_dir"])
         pre_check_resp = self.pre_check(master_node_list[0])
         LOGGER.debug("pre-check result %s", pre_check_resp)
         resp = self.deploy_cluster(master_node_list[0], self.deploy_cfg["k8s_dir"])
-        log_file = self.deploy_cfg['log_file']
-        matches = [re.compile(pat) for pat in self.patterns]
-        if not (m.match(resp[1]) for m in matches):
+        LOGGER.debug("deploy resp is %s", resp)
```

Comment:
The debug comment can be more elaborate to make it easy to understand? Also make the first letter in caps.